prelude/bigint.md

```diff
@@ -0,0 +1,52 @@
+BigInt is the type of mathematical integers.
+
+We assume all the usual arithmetic operations to be defined.
+Additionally, BigInt provides a few utility functions.
+
+```rust
+pub use specr::BigInt;
+
+impl BigInt {
```

Comment:
This won't work if `BigInt` is in a different crate, right?

I think long-term we'll want to move all of this to the specr crate. But for now this is fine.